docker-compose.yml

```diff
@@ -20,6 +20,9 @@ services:
       retries: 1
     volumes:
       - ./scripts/st2web-healthcheck.sh:/st2web-healthcheck.sh
+      # If you want to use a custom st2web config, edit ./files/config.js accordingly and
```

Comment:
Can we include a small section in the README as well? That'll be helpful for the broader community.